src/platform/riot/uart_channel.c

```diff
@@ -0,0 +1,161 @@
+#include "reactor-uc/platform/riot/uart_channel.h"
+#include "reactor-uc/logging.h"
+#include "reactor-uc/environment.h"
+#include "reactor-uc/serialization.h"
+#include "led.h"
+
+#define UART_CHANNEL_ERR(fmt, ...) LF_ERR(NET, "UARTSyncChannel: " fmt, ##__VA_ARGS__)
```

Comment:
```suggestion
#define UART_CHANNEL_ERROR(fmt, ...) LF_ERROR(NET, "UARTSyncChannel: " fmt, ##__VA_ARGS__)
```

Reply:
This is a change I introduced recently to stay compatible with reactor-c target properties